packages/mermaid/src/diagrams/class/classDb.ts

```diff
@@ -308,6 +320,7 @@ const setClickFunc = function (_domId: string, functionName: string, functionArg
   }
 
   const id = domId;
+  addClass(id);
   if (classes[id] !== undefined) {
```

Comment:
`classes[id] !== undefined` will never be falsey, since we just called `addClass()`.